clouditor-engine-core/src/main/java/io/clouditor/assurance/Rule.java

```diff
@@ -77,22 +67,15 @@
   @Column(name = "icon")
   private final String icon = "far fa-file-alt";
 
-  @JsonProperty
-  @ManyToMany
-  @LazyCollection(LazyCollectionOption.FALSE)
-  @JoinTable(
-      name = "rule_to_control",
-      joinColumns = @JoinColumn(name = "rule_id", referencedColumnName = "rule_id"),
-      inverseJoinColumns = @JoinColumn(name = "control_id", referencedColumnName = "control_id"))
-  private final List<Control> controls = new ArrayList<>();
+  @Embedded @JsonProperty private final List<String> controls = new ArrayList<>();
```

Comment:
same as above and loos of the relation to Contorl

Reply:
Yes, also intended because the way the controls and certification system currently works. Certification catalogs are imported through the UI, but rules exist beforehand, referencing a control ID, to a control which is not created yet. This is not the best system, however, I would stick to this right now and change the system later, i.e. to automatically import the catalogs at startup before the rules.